### PR TITLE
Add new sections to watsonx.ai prereq page for accessing watsonx.ai via TZ and IBM Cloud

### PR DESCRIPTION
**Pull request by maurelya**
Fixes | Relates to **[https://github.com/ibm-build-lab/VAD-VAR-Workshop/issues/227]**

Content changes:

- Any content updates here

Code changes:

- Any code changes here
http://localhost:8000/en/watsonx/watsonxai/100

### CODE REVIEW

**Review by ridgill — APPROVED** [review]
Looks good to me! Just waiting on @tkmonson to add any comments.

File: content/Watsonx/WatsonxAI/100.md [hunk 1]
```diff
@@ -15,6 +15,89 @@ toc: true
...
+
+### How to acquire a watsonx.ai instance from Techzone (TZ)
+
+
+To gain access to a watsonx.ai instance from TZ, navigate to this link [here](https://ibm.biz/WATSONX-AI-TZ). Once there:
+
+1. Select **Reserve now**
+     ![](images/100/reserve-now-btn.png)
+
+2. Change the Name of the reservation to *your name* watsonx.ai L3 SaaS (1).
```
Comment (tkmonson) [inline content/Watsonx/WatsonxAI/100.md:28]: This instruction includes 'L3' in the name, but the image does not have 'L3' in the name.

File: content/Watsonx/WatsonxAI/100.md [hunk 2]
```diff
@@ -15,6 +15,89 @@ toc: true
...
+
+13. When you log in, go to the menu bar on the top right and select the `itz-watsonx` account.
+
+      ![](images/100/itz-watsonx.png)
+
+14. In the search field, enter `watsonx`. The search will return a list similar to the following. Click `watsonx`.
+
+      ![](images/100/search-watsonx.png)
+
+15. You will now see the AI and data platform page. Click `Get started` from the watsonx.ai tile.
```
Comment (tkmonson) [inline content/Watsonx/WatsonxAI/100.md:69]: For some reason, the button says 'Launch' for me.

**Review by tkmonson — COMMENTED** [review]
LGTM, a couple of comments

File: content/Watsonx/WatsonxAI/100.md (hunk 2 above)
Comment (maurelya) [reply content/Watsonx/WatsonxAI/100.md:69]: This button may instead say **Launch** if you have provisioned a watsonx instance.

**Review by tkmonson — APPROVED** [review]
Changes look good!

**Pull request merged**